rllib/utils/pre_checks.py

```diff
@@ -1,9 +1,13 @@
 """Common pre-checks for all RLlib experiments."""
 import logging
+from typing import TYPE_CHECKING
 
 import gym
 from ray.rllib.utils.typing import EnvType
 
+if TYPE_CHECKING:
```

Comment:
why do you need this? I don't see a circular dep here?

Reply:
There is a circular import with the MultiAgentEnv class, since the utils module is imported when the environments are imported. My linter failed without this.